acceptance/Gemfile

```diff
@@ -1,21 +1,16 @@
 source "https://rubygems.org"
 
+gem "rake" # required to build some native extensions
```

Comment:
oh man. so many environment issues have trolled me with not having rake in Gemfiles. 👍